rootfs/bin/boot

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+echo "Powered by $POWERED_BY" > /usr/share/nginx/html/index.html
```

Comment:
Is this a bit of complexity that could be spared by just including that file outright with the correct contents already in it?  That would reduce the net effect of this boot script to just one command that could probably also just be moved up into the `CMD` section of the Dockerfile to spare us some more complexity.  But I suspect you have reasons for having done things this way...